packages/hooks/src/useDepositFiatAddress.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useEffect } from 'react';
+import { useRequest } from '@deriv/api';
+
+const useDepositFiatAddress = () => {
+    const { data, mutate, ...rest } = useRequest('cashier');
+    const deposit_iframe_url = typeof data?.cashier === 'string' ? data?.cashier : undefined;
```

Comment:
@farzin-deriv I think migrating the logic for dark-mode inside the hook is not possible because I am unable to wrap the hook with `observer` to observe the `is_dark_mode_on` 🤔 .

Reply:
@aum-deriv You don't need the `observer` in the hook 🙇🏻